R/cpp11.R

```diff
@@ -23,5 +22,21 @@ use_cpp11 <- function() {
     open = is_interactive()
   )
 
+  check_cpp_register_deps()
+
   invisible()
 }
+
+get_cpp_register_deps <- function() {
+  res <- desc::desc(package = "cpp11")$get_field("Config/Needs/cpp11/cpp_register")
```

Comment:
I added the packages needed for `cpp11::cpp_register()` in https://github.com/r-lib/cpp11/commit/bb6745fbc6ee4662c8fec28e6f9632c6b9855d69, not yet on CRAN, so I added cpp11 to the Remotes for now.

Reply:
That's a fantastic solution!